core/util/src/main/scala/net/liftweb/util/Props.scala

```diff
@@ -328,5 +378,18 @@ object Props extends Logger {
         Map()
     }
   }
+
+  private var prepended:List[java.util.Map[Object, Object]] = Nil
+  private var appended:List[java.util.Map[Object, Object]] = Nil
+  private var interpolators:List[java.util.Map[Object, Object]] = Nil
```

Comment:
Why wouldn't sys.env and sys.props be used instead? They both return immutable.Map.